pallets/gear-program/src/lib.rs

```diff
@@ -324,31 +304,6 @@ pub mod pallet {
         value: (BlockNumberFor<T>, H256)
```

Comment:
what about removing `PausedProgramStorage` here ?

Reply:
Removed the wrap type but kept storage itself for migration

Reply:
You can define storage inside migration if you need it there

Reply:
How I can do that? I am unable to find the docs for this. I need to again do `[frame_support::pallet] mod pallet`? 

Also sorry for again requesting review, misclicked